src/libostree/ostree-repo-checkout.c

```diff
@@ -579,22 +652,15 @@ checkout_tree_at (OstreeRepo                        *self,
                        (guint64)repo_dfd_stat.st_dev, (guint64)destination_stat.st_dev);
 
   /* Set the xattrs now, so any derived labeling works */
```

Comment:
Can we update this comment as well since we might have done labeling already above?